### ISSUE TITLE
doesn't work with `nim devel`

### ISSUE THREAD

**Issue by timotheecour** [brentp/nim-plotly#37]
i ran into this
```
nim --version
Nim Compiler Version 0.19.9 [MacOSX: amd64]
Compiled at 2019-01-25
Copyright (c) 2006-2018 by Andreas Rumpf

git hash: 44923a1b1f158732b13fb938c94c0b02a2463e6d
active boot switches: -d:release


git clone https://github.com/brentp/nim-plotly
cd nim-plotly

nimble develop
   Warning: Using env var NIM_LIB_PREFIX: /Users/timothee/git_clone//nim//Nim/
       Tip: 13 messages have been suppressed, use --verbose to show them.
     Error: Could not read package info file in /Users/timothee/git_clone/nim/nim-plotly/plotly.nimble;
        ...   Reading as ini file failed with:
        ...     Invalid section: .
        ...   Evaluating as NimScript file failed with:
        ...     /Users/timothee/git_clone/nim/Nim/lib/pure/bitops.nim(40, 39) Error: cannot evaluate 'sizeof' because its type is not defined completely, type: type T.
```

EDIT:  got it to work by a weird series of operations
```
choosenim 0.19.2
nimble install nim-plotly
choosenim /Users/timothee/git_clone/nim/Nim/ # restore devel
nimble develop # now worked
```

**Comment by Vindaar** [comment]
Hmm, weird. Good that it works now, but I'll check if I can reproduce that in the next couple of days.

**Comment by Vindaar** [comment]
Ok, I just stumbled on the same problem trying to bootstrap the compiler after updating devel from a mid November state to the current. So not related to plotly.

**Comment by timotheecour** [comment]
> So not related to plotly.

sure?

then maybe nimble ? (ie could become smart about requiring to update dependencies when needed; as `brew` does on OSX

**Comment by Vindaar** [comment]
Well, not sure sure, but on my machine I had to completely rebuild from csources, before I could compile anything again (got some weird `system.nim` `key:val` errors too). 

I don't use `choosenim`, so I'm not sure. I guess you somehow got into a similar state, which was fixed by going back to `0.19.2` before back to devel?
